validator/src/lib.rs

```diff
@@ -31,3 +31,4 @@ pub mod coretypes;
 pub mod database;
 pub mod proto;
 pub mod state;
+pub mod storage;
```

Comment:
It's probably better to follow the existing organization rather than invite discussion over a new reorganization.

Reply:
Yeah, I believe you are right.